src/_P036_FrameOLED.ino

```diff
@@ -734,7 +740,10 @@ boolean Plugin_036(uint8_t function, struct EventStruct *event, String& string)
           uint16_t PixLength = P036_data->display->getStringWidth(String(P036_data->DisplayLinesV1[LineNo - 1].Content));
 
           if (PixLength > 255) {
-            addHtmlError(String(F("Pixel length of ")) + String(PixLength) + String(F(" too long for line! Max. 255 pix!")));
+            String _error = F("Pixel length of ");
```

Comment:
Just curious, why using the _ prefix?
Last week I mentioned to @tonhuisman about using the _ prefix for member variables, so if you use them for temporary variables then I guess it could become confusing.

Reply:
Change in the next commit.